BAU: add readme for testing the IPV stub with the RP stub (#142)
govuk-one-login/orch-stubs: src/main/ipv-stub/README.md

### Testing with the RP stub

> Setup to point the RP stub when doing an identity journey to the Orch IPV stub.

Reviewer: ```suggestion
Setup to configure the RP stub to use the IPV stub:
```

1. Deploy [this branch](https://github.com/govuk-one-login/authentication-api/tree/refs/heads/ATO-867/connect-orch-ipv-stub-to-rp-stub) to orch dev. This branch changes the IPV flags to allow the dev environment and also points the `IPV URLs` to the IPV stub.

> 2. Change to sandpit client registry `1Dlz5rYheTqzZASRMmSBtgFIYgZlysnQ` to have the `IdentityVerificationSupported` flag turned on.
> 

Reviewer: ```suggestion
2. Locate the client registry DynamoDB item for sandpit RP stub (ClientID `1Dlz5rYheTqzZASRMmSBtgFIYgZlysnQ`), and manually change the value of `IdentityVerificationSupported` from 0 to 1.

```

This will allow you to test the Orch IPV stub with an identity journey executed from the sandpit RP stub.

> > **_NOTE:_**  Testing with the sandpit RP stub will lead you to an error page and not the userinfo results as the SPOT stub has not been setup in sandpit. Check the IPVcallbackHandler logs for more details.

Reviewer: ```suggestion
> **_NOTE:_**  Testing with the sandpit RP stub will lead you to an error page and not the userinfo results as the SPOT stub has not yet been created. Check IPVCallbackHandler logs for more details.
```
